packages/app/src/app/components/Preview/DevTools/Console/Message/index.js

```diff
@@ -50,6 +59,24 @@ function getMessage(message) {
   }
 
   if (message.arguments.every(argument => typeof argument === 'string')) {
+    if (message.logType === 'error') {
```

Comment:
Would also be good to put a comment here what the purpose of this code is. For new readers.